defi/src/storeTvlInterval/computeTVL.ts

```diff
@@ -13,6 +13,16 @@ type Balances = {
   [symbol: string]: number;
 };
 
+function prefixMalformed(address: string) {
+  const parts = address.split(':')
+  if (parts.length == 2) return false
+
+  if (parts.length == 1) return true
+  if (address.indexOf(':coingecko:') != -1) return true
+  if (parts[0] == parts[1]) return true
```

Comment:
this could be true in some cases like say in cardano, if we use `cardano` as gas token key, it will show up as `cardano:cardano`, thats why I suggested adding check if parts.length > 2 for this